integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioMonitoringExporter.java

```diff
@@ -119,24 +130,88 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
    * Verify server pods are in ready state and services are created.
    * Verify login to WebLogic console is successful thru istio ingress port.
    * Deploy a web application thru istio http ingress port using REST api.
-   * Access web application thru istio http ingress port using curl.
    * Deploy Istio provided Promethues
    * Verify Weblogic metrics can be processed via istio based prometheus
    */
   @Test
   @DisplayName("Create istio provided prometheus and verify "
       + "it can monitor Weblogic domain via weblogic exporter webapp")
-  public void testIstioPrometheus() {
-    assertDoesNotThrow(() -> setupIstioModelInImageDomain(), "setup for istio based domain failed");
-    int prometheusPort = getNextFreePort();
-    assertTrue(deployIstioPrometheus(domainNamespace, domainUid,
-        String.valueOf(prometheusPort)), "failed to install istio prometheus");
+  public void testIstioPrometheusViaExporterWebApp() {
```

Comment:
Please modify the test method javadoc to include monitoring exporter

Reply:
done